src/main/kotlin/sdksetup/SDKSetupFirstStepPanel.kt

```diff
@@ -0,0 +1,93 @@
+package sdksetup
+
+import OneSignalStep
+import OneSignalStepListener
+import com.intellij.openapi.project.Project
+import view.MultilineLabel
+import java.awt.GridBagConstraints
+import java.awt.GridBagLayout
+import javax.swing.JButton
+import javax.swing.JPanel
+
+class SDKSetupFirstStepPanel(
+    private val basePath: String,
+    private val project: Project,
+    private val stepListener: OneSignalStepListener
+) : JPanel(),
+    OneSignalStep {
+
+    private val controller = SDKSetupFirstStepController()
+    private val instructionString = """
+        OneSignal SDK needs the following changes in your build.gradle file
+
+        buildscript {
+
+            repositories {
+                ...
+                gradlePluginPortal()
+            }
+
+            dependencies {
+
+               classpath 'com.onesignal:onesignal-gradle-plugin:[0.8.1, 0.99.99]'
```

Comment:
Update to latest version, which is `0.14.0`. Also we should probably have a shared definition for this version string, since it is used in multiple spots.